### PR TITLE
Change v1alpha1 to v1 in Serverless docs code samples

### PR DESCRIPTION
**Pull request by abrennan89**
Should apply for OCP 4.3, 4.4 and 4.5.

### CODE REVIEW

File: modules/configuring-scale-bounds-knative.adoc [hunk 1]
```diff
@@ -16,6 +16,10 @@ maxScale:: If the `maxScale` annotation is not set, there will be no upper limit
 
 [source,yaml]
 ----
+ApiVersion: serving.knative.dev/v1
```
Comment (bbrowning) [inline modules/configuring-scale-bounds-knative.adoc:19]: This should be a lowercase `a`

**Review by bbrowning — COMMENTED** [review]
Just one nit on the capitalization - otherwise, looks good!

File: modules/configuring-scale-bounds-knative.adoc (hunk 1 above)
Comment (cardil) [inline modules/configuring-scale-bounds-knative.adoc:19]: I'd not adding this here. This was a YAML fragment. By adding this it seems like this is fully working example, but it isn't.

**Review by cardil — CHANGES_REQUESTED** [review]

**Comment by bmcelvee** [comment]
LGTM - I'll merge and cherry pick as soon as I get a few cherry picks in the queue merged :)

**Pull request merged**